packages/console/src/components/Button/index.tsx

```diff
@@ -1,13 +1,14 @@
 import { I18nKey } from '@logto/phrases';
 import classNames from 'classnames';
-import React, { HTMLProps } from 'react';
+import React, { HTMLProps, ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
 
 import * as styles from './index.module.scss';
 
-type Props = Omit<HTMLProps<HTMLButtonElement>, 'type' | 'size' | 'title'> & {
+type Props = Omit<HTMLProps<HTMLButtonElement>, 'type' | 'size' | 'icon' | 'title'> & {
   htmlType?: 'button' | 'submit' | 'reset';
-  title: I18nKey;
+  icon?: ReactNode;
```

Comment:
Maybe we can defined a `Icon` component with props like size, can do it later or take a discussion.

Reply:
let's add this into discussion

Reply:
LOG-1888